src/ui/public/filter_bar/push_filter.js

```diff
@@ -7,10 +7,26 @@ export default function () {
       // Hierarchical and tabular data set their aggConfigResult parameter
       // differently because of how the point is rewritten between the two. So
       // we need to check if the point.orig is set, if not use try the point.aggConfigResult
-      let filters = _.clone($state.filters || []);
-      let pendingFilter = { meta: { negate: negate, index: index }};
+      const filters = _.clone($state.filters || []);
+      const pendingFilter = { meta: { negate: negate, index: index }};
       _.extend(pendingFilter, filter);
-      filters.push(pendingFilter);
+      const filterKey = _.keys(filter)[0];
+      let filterAdded = false;
+
+      filters.forEach(function (filt) {
+        const isDisabled = filt.meta.disabled;
+        const isSameFilterType = filt.hasOwnProperty(filterKey);
+        if (!isDisabled && isSameFilterType) {
+          _.assign(pendingFilter.meta, filt.meta); // be sure to keep any meta data
```

Comment:
Hmm, so should I check and make sure they're the same index pattern?I figured if checking to make sure they're the same doesn't matter, neither does overwriting it.

Also, what do I do in the ui to make it change?